tokens/src/components/Tooltip.json

```diff
@@ -1,38 +1,50 @@
 {
-  "tooltip": {
-    "font": {
-      "size": { "value": "{font.size.sm.value}", "source": "$tooltip-font-size" }
-    },
-    "max-width": { "value": "200px", "source": "$tooltip-max-width" },
-    "color": {
+  "typography": {
+    "tooltip": {
+      "font-size": { "value": "{font.size.sm.value}", "source": "$tooltip-font-size" }
+    }
+  },
+  "color": {
+    "tooltip": {
       "base": { "value": "{color.white.value}", "source": "$tooltip-color" },
```

Comment:
I think we should switch key to `text` here, because this color applies only to text I think